build/make-olm.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+CSV_VERSION="${1:-0.1.0}"
```

Comment:
I intentionally use 0.0.0 as default to have a "known bad"/impossible version, and not something that could look right